docs-rtd/howto/manage-migrations.md

```diff
@@ -0,0 +1,101 @@
+(manage-migrations)=
+
```

Comment:
I'd remove the empty line. (Our current style doesn't leave empty lines between an anchor and the heading it's anchoring.)